src/download_model.py

```diff
@@ -71,6 +71,24 @@ def main():
                    input_location=os.path.join('outputs', 'best_model.ckpt'),
                    output_location=REPO_DIR / 'models/depthmapmultiartifactlatefusion')
 
+    RUN_IDS = ['q1-ensemble-warmup_1610544610_eb44bfe2', 'q1-ensemble-warmup_1610547587_7ca932c3',
```

Comment:
Where i should write RUN_IDs at the top?

Reply:
Global variables are often defined in config files or on the top of the module (also cross-check with our other code)